identicon.js

```diff
@@ -118,8 +118,13 @@
             ];
         },
 
-        toString: function(){
-            return this.render().getBase64();
+        toString: function(type){
```

Comment:
Rather than `type` as a string argument, how about `raw` as a boolean. If you pass `raw=true` then you get the xml for SVGs and `getDump()` for PNGs.